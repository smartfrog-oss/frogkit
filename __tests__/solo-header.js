import { mount } from 'vue-test-utils'
import Soloheader from '@/components/solo-header'

describe('Soloheader component', () => {
  it('should render component and match snapshot', () => {
    const wrapper = mount(Soloheader)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
