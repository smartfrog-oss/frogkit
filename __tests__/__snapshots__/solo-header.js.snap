// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Soloheader component should render component and match snapshot 1`] = `undefined`;
